src/controls/uploadFiles/components/DocumentList/DocumentList.tsx

```diff
@@ -0,0 +1,189 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import * as React from 'react';
+
+import strings from 'ControlStrings';
+import { useAtom } from 'jotai';
+import {
+  isEqual,
+  pullAllWith,
+} from 'lodash';
+import {
```

Comment:
Can you please import from office-ui-fabric-react/lib/xxxxx

Reply:
done